server/internal/app-config/config.go

```diff
@@ -147,6 +150,22 @@ func (c *Config) repoManager() error {
 			RoundStoreConfig: []interface{}{c.DbDir, logger},
 			VtxoStoreConfig:  []interface{}{c.DbDir, logger},
 		})
+	case "sqlite":
+		var database *sql.DB
+		database, err = sqlitedb.OpenDB(c.DbDir + "/sqlite.db")
+		if err != nil {
+			return err
+		}
+
+		svc, err = db.NewService(db.ServiceConfig{
+			EventStoreType: c.DbType,
+			RoundStoreType: c.DbType,
+			VtxoStoreType:  c.DbType,
+
+			EventStoreConfig: []interface{}{c.DbDir, log.New()},
+			RoundStoreConfig: []interface{}{database},
+			VtxoStoreConfig:  []interface{}{database},
```

Comment:
we can just pass the same config we use for badger in this case:
```suggestion
			RoundStoreConfig: []interface{}{c.DbDir, logger},
			VtxoStoreConfig:  []interface{}{c.DbDir, logger},
```

Reply:
`c.DbDir`: isn't better to open the DB only one time and then pass the `*sql.Db` to repositories ? To minimize the number of "open file operations" ? 

I don't see the point to pass the logger ? Do you mean we should use a custom logger to print some "Debug" logs ?

Reply:
> c.DbDir: isn't better to open the DB only once and then pass the *sql.Db to repositories ? If I pass the path instead, I will execute 2 times the OpenDb function (expensive ?)

totally agree, but let's do it in the repo manager, not here in config.. we did the same for badger, the change is as simple as copy-pasting the existing code (mostly)